cocos/tween/actions/action-interval.ts

```diff
@@ -809,6 +862,22 @@ export class Spawn extends ActionInterval {
         }
         return null;
     }
+
+    isUnknownTime (): boolean {
+        const one = this._one;
+        const two = this._two;
+
+        if (one == null || two == null) return false;
+
+        const isOneUnknownTime = one.isUnknownTime();
+        const isTwoUnknownTime = two.isUnknownTime();
+
+        if (isOneUnknownTime || isTwoUnknownTime) {
+            if (isOneUnknownTime && isTwoUnknownTime) return true;
+            else if (this._finished) return true;
```

Comment:
Don't understand the logic, why return true if this is finished.?

Reply:
Suppose that there are two action A and B in Spawn, A is an action with unknown duration (0s) while B is an action with constant duration (e.g. 1 second), when 1 second passes, B will be done, so the whole Spawn is done, but A is not resovled, so the Spawn is still in `unknown duration` state.

Reply:
In this situation, i think it should return true event this._finished is false. May be can change the code like this
```typescript
return (isOneUnknownTime || isTwoUnknownTime);
```

Reply:
`(isOneUnknownTime || isTwoUnknownTime);` means one of the actions in Spawn is unknown duration, another one is not. Still take the above example:
- Spawn starts
- A, B update at the same time, A is `unknown duration`, B is fixed duration (1 second) 
- Suppose 0.5 second was passed, A keeps its `unknown duration` state, B `is not` done. B needs to be continued get `update` with a `t` increased. So the `Spawn` is `not` in `unknown duration` state.
- While 1 seconds was passed, A still keeps its `unknown duration` state, B `is` done. B doesn't needs to be continued get `update` now. But A is still waiting, so the whole `Spawn` needs to be waiting too. And the `Spawn` is in `unknown duration` state to avoid `elasped time` being added `dt`.